tests/unit/codegen/sdk/typescript/move_symbol_to_file/test_move_tsx_to_file.py

```diff
@@ -127,26 +129,79 @@ def test_remove_unused_exports(tmpdir) -> None:
     return (<Wrapper components={[MainComponent, SharedComponent]}/>)
 }
 """
-    misc_filename = "misc.tsx"
     # language=typescript jsx
-    misc_content = """
+    MISC_CONTENT = """
 export { UnusedComponent } from 'Component'
 function Helper({ props }: HelperProps) {}
 
 export { Helper }
 """
-    import_filename = "import.tsx"
     # language=typescript jsx
-    import_content = """
+    IMPORT_CONTENT = """
 import { UnusedComponent } from 'misc'
 """
 
-    files = {src_filename: src_content, adj_filename: adj_content, misc_filename: misc_content, import_filename: import_content}
+    # ========== [ AFTER ] ==========
+    # language=typescript jsx
+    EXPECTED_SRC_CONTENT = """
+import { SubComponent } from 'new';
+
+export default function MainComponent() {
+  const [state, setState] = useState<StateType | null>()
+  return (<div>
+            <div>
+                <SubComponent/>
+            </div>
+          </div>)
+}
+
+export function UnusedComponent({ props }: UnusedProps) {
```

Comment:
Refactoring it this way is good, can we split this out so we can see the before/after of the new feature?